spec/models/measure_spec.rb

```diff
@@ -1224,4 +1224,54 @@
 
     it { expect(measure.meursing_measures).to eq(meursing_measures) }
   end
+
+  shared_context 'with regulation measures' do
+    around do |example|
+      TimeMachine.now { example.run }
+    end
+
+    before do
+      create(:base_regulation, base_regulation_id: 'R9726580', base_regulation_role: 1)
```

Comment:
There are three types of base regulation (as indicated by BASE_REGULATION_ROLES). I'm filtering on these to make sure we get the correct regulation end date - we were getting nil because we were indifferent to the role and the role for base regulation with no end date came first